node-group-project/src/components/PetsContainer.js

```diff
@@ -1,24 +1,35 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import _ from "lodash"
+import { Link } from "react-router-dom"
 
 const PetsContainer = (props) => {
   const [petTypes, setPetTypes] = useState([]);
 
   useEffect(() => {
-    fetch('http://localhost:3000/pets')
-      .then(result => setPetTypes(result.rows))
+    fetch('http://localhost:3000/api/v1/pettypes')
```

Comment:
Is this path the correct path? Your endpoint would have to have the http://localhost:3000 to work. I recommend simplifying it if possible.